ToDo.swift

```diff
@@ -0,0 +1,16 @@
+//
+//  ToDo.swift
+//  ToDoList
+//
+//  Created by Larry Brewer on 3/26/18.
+//  Copyright © 2018 Larry Brewer. All rights reserved.
+//
+
+import Foundation
+
+class ToDo {
+    
+    var description = ""
+    var status = false
```

Comment:
if using a boolean, you should name status `completed`
if using a status enum, status is a fine name.